### PR TITLE
Support dynamic changing log level in status manager

### PR DESCRIPTION
**Pull request by Okabe-Rintarou-0**
**What type of PR is this?**
feature

**What this PR does / why we need it**:
Support dynamic changing log level in status manager
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by Okabe-Rintarou-0** [comment]
Example:

image

**Comment by Okabe-Rintarou-0** [comment]
maybe we can also add some shortcuts in command, like:
+ `kmesh-daemon logs`: print logs
+ `kmesh-daemon logger --name default --set level=info`

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
> maybe we can also add some shortcuts in command, like:
>
> kmesh-daemon logs: print logs
> kmesh-daemon logger --name default --set level=info: set the specified logger's level

How about ` kmesh-daemon log --set default:debug`, if no --set specified, treat it as get log level

File: pkg/status/status_server.go [hunk 1]
```diff
@@ -130,6 +141,79 @@ func (s *Server) bpfAdsMaps(w http.ResponseWriter, r *http.Request) {
...
+	if err != nil {
+		log.Errorf("Failed to marshal logger info: %v", err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write(data)
+}
+
+func (s *Server) setLoggerLevel(w http.ResponseWriter, r *http.Request) {
```
Comment (LiZhenCheng9527) [inline pkg/status/status_server.go:182]: Can you add UT to the added functions?

Comment (Okabe-Rintarou-0) [reply pkg/status/status_server.go:182]: sure

**Comment by Okabe-Rintarou-0** [comment]
> > maybe we can also add some shortcuts in command, like:
> > kmesh-daemon logs: print logs
> > kmesh-daemon logger --name default --set level=info: set the specified logger's level
> 
> How about ` kmesh-daemon log --set default:debug`, if no --set specified, treat it as get log level

yes, it'd be better. I will work on it.

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
/lgtm

Please support the subcmd in a following up pr

**Pull request merged**